### PR TITLE
Updates to readme, formatting.

### PR DESCRIPTION
**Pull request by lisakoivu**
Overhauling rds postgres alarms.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,87 @@
...
-AWS RDS postgres specific alarm definitions (not Aurora).
+This repo defines AWS RDS postgres specific alarm definitions (not Aurora). These alarms monitor postgres replication between master and replica.
 
-Threshold alarms are included. Actions can be turned off via variable anomaly_actions_enabled. 
+Each alarm listed below is created by default. Creation can be disabled by setting the associated alarm create parameter to false. For example, to not create the alarm replica_db_load_anomaly, set the parameter create_replica_db_load_anomaly_alarm to false.
+
+Threshold alarms are included. Anomaly alarm creation can be disabled by setting the variable create_anomaly_alarms to false.
+
+
+master_checkpoint_lag_anomaly_alarm
```
Comment (v47) [inline README.md]: can you make it a bullet list?

File: master.tf [hunk 2]
```diff
@@ -0,0 +1,1095 @@
+#------------------------------------------------------------------------------
+# Single Instance master threshold alarms
+resource "aws_cloudwatch_metric_alarm" "master_cpu_utilization_high" {
+  count                     = var.create_master_cpu_utilization_high_alarm ? 1 : 0
+  alarm_name                = "${var.db_instance_id}_cpu_utilization_high_static"
+  comparison_operator       = "GreaterThanThreshold"
+  evaluation_periods        = local.thresholds["CPUUtilizationEvaluationPeriods"]
+  metric_name               = "CPUUtilization"
+  namespace                 = "AWS/RDS"
```
Comment (v47) [inline master.tf]: you can create a namespace local varible.
something like:
```
locals{
  cloudwatch_namespace = "AWS/RDS"
}
```

File: master.tf [hunk 3]
```diff
@@ -0,0 +1,1095 @@
...
+      stat        = "Maximum"
+
+      dimensions = {
+        DBInstanceIdentifier = var.db_instance_id
+      }
+    }
+  }
+}
+
+
```
Comment (v47) [inline master.tf]: unnecessary newlines.

File: master.tf [hunk 4]
```diff
@@ -0,0 +1,1095 @@
...
+      period      = "60"
+      stat        = "Maximum"
+
+      dimensions = {
+        DBInstanceIdentifier = var.db_instance_id
+      }
+    }
+  }
+}
+
```
Comment (v47) [inline master.tf:396]: unnecessary newlines.

File: README.md (hunk 1 above)
Comment (lisakoivu) [reply README.md]: done

File: master.tf (hunk 2 above)
Comment (lisakoivu) [reply master.tf]: done

File: master.tf (hunk 4 above)
Comment (lisakoivu) [reply master.tf:396]: extra carriage returns removed

**Review by v47 — APPROVED** [review]

**Pull request merged**